### ISSUE TITLE
Mobilenetv3 quantitative model accuracy reproduction

### ISSUE THREAD

**Issue by itminner** [tensorflow/models#8664]
I’m trying to reproduce the Mobilenetv3 quantization process, using mobilenetv3 large dm=1(top1: 75.2% in imagenet 2012). My int8 quantitative model top1 accuracy is only 71%.
How can I reproduce the int8 model accuracy in the document, which top1 accuracy is 73.9%.
<img width="941" alt="图片" src="https://user-images.githubusercontent.com/16641575/84460381-5b9b4700-ac9c-11ea-872d-089bebd74821.png">

Can you provide detailed reproduction steps, such as learning rate and batchsize parameters.
Do I need to skip some layers and not quantify them?
